cisco_umbrella_reporting/plugin.spec.yaml

```diff
@@ -0,0 +1,119 @@
+  attackers infrastructures and predict future threats
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/cisco_umbrella_reporting
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.cisco.com
+tags:
+- cisco
+- umbrella
+- reporting
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [cisco, umbrella, reporting]
+  features: []
+connection:
+  api_key:
+    title: API Key
+    description: Cisco Umbrella API key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  api_secret:
+    title: API Secret Key
+    description: Cisco Umbrella API secret key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  organization_id:
+    title: Organization ID
+    description: ID of your Cisco Umbrella organization
+    type: string
+    required: true
+    example: 2961483
+types:
+  record:
+    originid:
+      title: Origin ID
+      description: Numerical identifier for the identity making the request
+      type: integer
+      required: false
+    internalIp:
+      title: Internal IP
+      description: Internal IP address of the identity making the request
+      type: string
+      required: false
+    externalIp:
+      title: External IP
+      description: External IP address of the identity making the request
+      type: string
+      required: false
+    originLabel:
+      title: Origin Label
+      description: Human-readable name for the identity, matching the one seen in the dashboard
+      type: string
+      required: false
+    categories:
+      title: Categories
+      description: Which categories, if any, the destination for which this request was made falls into
+      type: "[]string"
+      required: false
+    originType:
+      title: Origin Type
+      description: Identity type (such as network, roaming computer, AD User, etc)
+      type: string
+      required: false
+    actionTaken:
+      title: Action Taken
+      description: Either Blocked or Allowed
+      type: string
+      required: false
+    datetime:
+      title: Datetime
+      description: Datetime at which the event occurred, in UTC
+      type: string
+      required: false
+    destination:
+      title: Destination
+      description: Destination to which this request was made
+      type: string
+      required: false
+    tags:
+      title: Tags
+      description: Which tags, if any, the destination for which this request was made falls into
+      type: "[]string"
+      required: false
+actions:
+  get_domain_visits:
+    title: Get Domain Visits
+    description: Get a list of computers that visited a domain within the last 24 hours
+    input:
+      most_recent:
+        title: Most Recent
+        description: Set to true to get the most recent DNS requests for a domain
```

Comment:
Do we know what the difference is between setting this to true and false? The action is for last 24 hours which seems recent to me. What does most recent enable compared to not setting it?

Reply:
@jonschipp 
Here are used two endpoints: 
/reporting-destinations-most-recent-requests
/security-activity-report
First get 500 records in last 24 hours and later. So it should get 500 data from week in example. 
The second one get 500 records but only from 24 hours.

Reply:
@r7-mgorny  Thanks can you document taht in the descriptions and in the help.md?

Reply:
Documented.